### PR TITLE
move bswap and tsc to i386

### LINKED ISSUE #313: Move bswap and tsc intrinsics to the i386 module
They are currently in the i586 module but both gcc and clang define them in the i386 modules: 

https://github.com/gcc-mirror/gcc/blob/master/gcc/config/i386/ia32intrin.h

### PR DESCRIPTION
**Pull request by llogiq**
This fixes #313

### CODE REVIEW

**Comment by gnzlbg** [comment]
@llogiq Thanks! This LGTM.

Could you fix the RUSTFMT build bot? Is complaining about an extra line somewhere. 

I'll merge this afterwards :)

File: coresimd/src/x86/i386/mod.rs [hunk 1]
```diff
@@ -7,3 +7,10 @@ pub use self::eflags::*;
 mod fxsr;
 #[cfg(dont_compile_me)] // TODO: need to upstream `fxsr` target feature
 pub use self::fxsr::*;
+
+mod bswap;
+pub use self::bswap::*;
+
+mod rdtsc;
+pub use self::rdtsc::*;
+
```
Comment (gnzlbg) [inline coresimd/src/x86/i386/mod.rs]: Looks like rustfmt does not like the whitespace here :(

**Comment by llogiq** [comment]
Will do.

**Comment by gnzlbg** [comment]
Thanks again!

**Comment by llogiq** [comment]
You're welcome. I'm looking forward to contribute more, however limited my time is. :+1:

**Pull request merged**